admin/database/database.go

```diff
@@ -352,21 +354,25 @@ type InsertUsergroupOptions struct {
 
 // UserAuthToken is a persistent API token for a user.
 type UserAuthToken struct {
-	ID           string
-	SecretHash   []byte    `db:"secret_hash"`
-	UserID       string    `db:"user_id"`
-	DisplayName  string    `db:"display_name"`
-	AuthClientID *string   `db:"auth_client_id"`
-	CreatedOn    time.Time `db:"created_on"`
+	ID                 string
+	SecretHash         []byte    `db:"secret_hash"`
+	UserID             string    `db:"user_id"`
+	DisplayName        string    `db:"display_name"`
+	AuthClientID       *string   `db:"auth_client_id"`
+	RepresentingUserID string    `db:"representing_user_id"`
+	ExpirationTS       time.Time `db:"expiration_ts"`
+	CreatedOn          time.Time `db:"created_on"`
```

Comment:
1. RepresentingUserID should be `*string`
2. `ExpirationTS` should be `ExpiresOn`, it should be `*time.Time` and come after `CreatedOn`